cluster/environments/azure-simple/main.tf

```diff
@@ -1,61 +1,29 @@
-provider "azurerm" {
-  subscription_id = "${var.subscription_id}"
-  client_id       = "${var.service_principal_id}"
-  client_secret   = "${var.service_principal_secret}"
-  tenant_id       = "${var.tenant_id}"
+terraform {
```

Comment:
We shouldn't require a backend for `azure-simple` IMO.  It requires extra setup / time and more problems for our CI system.  It isn't very hard to setup a backend and some terraform code to provision it + docs is enough IMO.  If we do want to keep it somewhere (i.e. `azure-advanced`) then we should provision the storage account for the backend as an option in our repo.

Reply:
Agreed - I will remove this.